### PR TITLE
[Frontend] Update TTL in metadata cache path

### PR DESCRIPTION
**Pull request by snalli**
This patch updates the ttl of the received metadata so that it can
be compared with the cached metadata. Apparently, the received metadata
isn't up-to-date and must be fixed upon receipt.

### CODE REVIEW

**Review by SophieGuo410 — APPROVED** [review]

**Review by Arun-LinkedIn — APPROVED** [review]

File: ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java
Comment (justinlin-linkedin) [inline ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java:1870]: nit: please format this file.

File: ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java
Comment (justinlin-linkedin) [inline ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java:1501]: Sorry I was sick a couple days ago and didn't have enough time to review the unit test code, but I think the name of "AmbryCacheMetrics" is confusing, especially on "Metrics" part. 
We have a convention to use "Metric" name when it's actually the metrics, like histogram, timer, counter etc.. This object is not metrics, it's more of stats. I suggest we change the name to something like AmbryCacheStats? What do you think?

File: ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java
Comment (snalli) [reply ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java:1870]: this file or this line ?

File: ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java
Comment (snalli) [reply ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java:1501]: I am fine with name but I'd prefer to do it in a separate follow up patch rather than this one. There would be many renames across many files.

Comment (snalli) [reply ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java:1501]: I mean I am fine with renaming, but in a follow up patch.

File: ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java
Comment (snalli) [reply ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java:1870]: done

**Comment by snalli** [comment]
Got two approvals. If the maintainers can merge, I could deploy.

**Review by justinlin-linkedin — APPROVED** [review]

**Pull request merged**
